AssemblerLib/LocalToGlobalIndexMap.h

```diff
@@ -86,27 +95,30 @@ class LocalToGlobalIndexMap
     /// \attention The passed mesh_component_map is in undefined state after
     /// this construtor.
     explicit LocalToGlobalIndexMap(
-        std::vector<MeshLib::MeshSubsets*> const& mesh_subsets,
+        std::vector<MeshLib::MeshSubsets*>&& mesh_subsets,
+        std::vector<std::size_t> const& original_indices,
         std::vector<MeshLib::Element*> const& elements,
-        AssemblerLib::MeshComponentMap&& mesh_component_map,
-        AssemblerLib::ComponentOrder const order);
+        AssemblerLib::MeshComponentMap&& mesh_component_map);
 
     template <typename ElementIterator>
     void
     findGlobalIndices(ElementIterator first, ElementIterator last,
-        std::size_t const mesh_id, AssemblerLib::ComponentOrder const order);
+        std::size_t const mesh_id,
+        const unsigned component_id, const unsigned comp_id_write);
 
 private:
-    std::vector<MeshLib::MeshSubsets*> const& _mesh_subsets;
+    std::vector<MeshLib::MeshSubsets*> const _mesh_subsets;
     AssemblerLib::MeshComponentMap _mesh_component_map;
 
-    /// Vector contains for each element a vector of global row/or entry indices
-    /// in the global stiffness matrix or vector
-    std::vector<LineIndex> _rows;
+    using Table = Eigen::Matrix<LineIndex, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>;
```

Comment:
It is possible to place this alias some where outside of this file, e.g. a a file like ``MathLib/LinAlg/MatrixTools.h`` for comment access? Under such consideration, we can replace the eigen with other third party algebraic package easily.